scripts/extract_hash.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/expect
+
+set timeout 60
+cd $::env(KEYSTONE_DIR)
+spawn ./scripts/run-hifive-qemu.sh
```

Comment:
This should use the more general:
`make qemu`
which is provided by the top-level Makefile (upcoming changes would otherwise break it  :P)

Reply:
The extract .sh script does not do anything anymore. I will have to fix that in the get attestation script.

Reply:
This will be updated once the keystone repo has fixed what `make qemu` does to launch qemu since the current build relies on the hifive build.